ROBT 3341 - SCARA/scara_control.c

```diff
@@ -107,10 +111,22 @@ void moveTCP(void) {
 # Last Modified: September 30, 2025
 # ---------------------------------------------------------------------------*/
 void limpUnlimp(void) {
-//Mine
-}
 
+printf("Enter a 1 to put the Scara into Limp Mode. Or any other key to proceed: \n");
+scanf("%d" &limp);
 
+if(limp)
+{
+	disableScaraMotors();
+	return;

```

Comment:
These returns are unnecessary, there's nothing left for the function to do and it will return naturally after this if/else is resolved.